kentikapi/client.go

```diff
@@ -174,58 +203,38 @@ func makeRetryingClientConfig(c Config) httputil.ClientConfig {
 	}
 }
 
-//nolint: gosec
-func makeConnForSynthetics(c Config) grpc.ClientConnInterface {
-	if c.SyntheticsGRPCAPIURL == "" {
-		c.SyntheticsGRPCAPIURL = syntheticsGRPCAPIURL
-		config := &tls.Config{}
-		conn, err := grpc.Dial(
-			syntheticsGRPCAPIURL,
-			grpc.WithTransportCredentials(credentials.NewTLS(config)),
-			grpc.WithUnaryInterceptor(c.createUnaryInterceptor()),
-		)
-		if err != nil {
-			log.Printf("cannot connect to api url for synthetics: %v", err)
-		}
-		return conn
-	}
-	conn, err := grpc.Dial(c.SyntheticsGRPCAPIURL, grpc.WithInsecure())
-	if err != nil {
-		log.Printf("cannot connect to api url for synthetics: %v", err)
-	}
-	return conn
-}
-
-//nolint: gosec
-func makeConnForCloudExport(c Config) grpc.ClientConnInterface {
-	if c.CloudExportGRPCAPIURL == "" {
-		c.CloudExportGRPCAPIURL = cloudExportGRPCAPIURL
-		config := &tls.Config{}
+func (c Config) makeConnForGRPC(hostPort string) (grpc.ClientConnInterface, error) {
+	if !c.DisableTLS {
+		config := &tls.Config{MinVersion: tls.VersionTLS13}
 		conn, err := grpc.Dial(
-			cloudExportGRPCAPIURL,
+			hostPort,
 			grpc.WithTransportCredentials(credentials.NewTLS(config)),
 			grpc.WithUnaryInterceptor(c.createUnaryInterceptor()),
 		)
 		if err != nil {
-			log.Printf("cannot connect to api url for cloudexport: %v", err)
+			return nil, fmt.Errorf("cannot connect to api: %v", err)
 		}
-		return conn
+		return conn, nil
 	}
-	conn, err := grpc.Dial(c.CloudExportGRPCAPIURL, grpc.WithInsecure())
+	conn, err := grpc.Dial(
+		hostPort,
+		grpc.WithInsecure(),
+		grpc.WithUnaryInterceptor(c.createUnaryInterceptor()),
+	)
 	if err != nil {
-		log.Printf("cannot connect to api url for cloudexport: %v", err)
+		return nil, fmt.Errorf("cannot connect to api: %v", err)
 	}
-	return conn
+	return conn, err
 }
 
 func (c Config) createUnaryInterceptor() grpc.UnaryClientInterceptor {
```

Comment:
I forgot to comment on that earlier:
```suggestion
func (c Config) makeAuthInterceptor() grpc.UnaryClientInterceptor {
```

`auth`: It is important for the name to say what the function does, not only the type it returns. 
`make`: personal preference - I see make usage in this file

Reply:
Done